src/main/java/p3/sramsbackend/PromMetric.java

```diff
@@ -0,0 +1,32 @@
+package p3.sramsbackend;
+
+public class PromMetric {
+
+    private String __name__;
+    private String instance;
+    private String job;
+
+    public String get__name__() {
```

Comment:
Kunne godt tænke mig et andet metode navn her 🤔
```suggestion
    public String getName() {
```